src/modules/auth/components/FormSignUp.tsx

```diff
@@ -0,0 +1,41 @@
+import { Input } from './Input';
+import { useTranslation } from 'react-i18next';
+
+export function FormSignUp(): JSX.Element {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <h2 className="text-4xl">Sign Up</h2>
+      <form className="flex flex-col space-y-1">
```

Comment:
I like that u used the `<form>` element... I hope it is a good sign you will write semantic web-friendly components.